site/faq.md

```diff
@@ -48,4 +48,22 @@ Provide Flux with the registry credentials. See
 
 Flux polls image registries every 5 minutes by default. You can change
 this, but beware that registries may throttle and even blacklist
-over-eager clients (like FLux in this scenario).
+over-eager clients (like Flux in this scenario).
+
+### How do I use my own deploy key?
+
+Flux uses a k8s secret to hold the git ssh deploy key. It is possible to
+provide your own.
+
+First delete the secret (if it exists):
+
+`kubectl delete secret flux-git-deploy`
+
+Then create a new secret named `flux-git-deploy` and using your key as the content of the secret:
```

Comment:
Just noticed this. Should either say "..., using your key..." or "...and use your key..."